src/mock/class_addressGenerator.py

```diff
@@ -66,7 +66,7 @@ class FakeAddressGenerator(StoppableThread):
     def stopThread(self):
         try:
             queues.addressGeneratorQueue.put(("stopThread", "data"))
-        except:
+        except:  # noqa:E722
```

Comment:
probably `queue.Full`

Reply:
Updated as we didi it for src/class_addressGenerator.py